math/divisor.cpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ * @brief Algorithms to find divisor of all natural numbers.
+ * @author [Tee Chia Suen](https://github.com/ChiaSuen)
+ */
+
+#include <cassert>  /// for assert
+#include <iostream>
+#include <vector>  //for declaring undefined size array
+
+/**
+ * @namespace <check from other files in this repo>
+ */
```

Comment:
```suggestion
/**
 * @namespace math
 * @brief Mathematical algorithms
 */
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant